packages/token-metadata/src/types.ts

```diff
@@ -59,9 +62,9 @@ export interface TokenMeta {
   coinGeckoId: string
 
   ibc?: IbcTokenMeta
+  ibcs?: IbcTokenMeta[]
   spl?: SplTokenMeta
-  cw20?: Cw20TokenMeta
-  cw20s?: Cw20TokenMetaWithSource[] // When there are multiple variations of the same CW20 token
+  cw20s?: Cw20TokenMeta[]
```

Comment:
Why do we have`ibc?:` but we dont have `cw20?:`?

Reply:
I've made it so that cw20 meta is merged to the main token meta body, we can directly use it as per any other tokens in our implementation.

So instead of `token.cw20.decimals`, we can use `token.decimals` directly. 

For cw20 source, we can do this:
```
(token as Cw20Token).soruce
```
For ibc, because of the list of custom fields, it will be cleaner to introduce a separate type as the format we have now:
`token.ibc.baseDenom`

Reply:
I think we should keep it consistent, either have both ibc and cw20 (which I'd prefer) or both merged

Reply:
sure, will revert